mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/CassandraMailboxManager.java

```diff
@@ -123,4 +123,10 @@ protected StoreMessageManager createMessageManager(Mailbox mailboxRow, MailboxSe
             getMessageIdFactory());
     }
 
+    @Override
+    public void createMailbox(MailboxPath mailboxPath, MailboxSession mailboxSession) throws MailboxException {
+        super.createMailbox(mailboxPath, mailboxSession);
+        // Heal potential mailbox duplication
```

Comment:
comments like that is a sign of failure : why does the method you call not named "healMailboxDuplication" ?